### DISCUSSION TITLE
Make dial-up noises while connecting through modem

### DISCUSSION THREAD

**Discussion by floridainsummer505** [86Box/86Box#4820] [Ideas]
Make dial-up noises while connecting through modem

**Comment by TheVirtualComputer** [comment]
Wouldn't this require emulation of an actual phone line?

**Reply by garoto** [reply]
I assumed OP was asking for a short PCM loop of modem sounds.

**Reply by artemchik12** [reply]
yep

**Reply by floridainsummer505** [reply]
yes

**Reply by Torinde** [reply]
fax/modem sound output would be great - both for dialing, handshake and for the communication itself (e.g. like in the past happened that "somebody picks up the phone while the computer is connected - listens to the audio flow between the two modems")

Next level: connect the emulated modem audio output to a real modem on the other side ([computer speaker/mic placed onto regular phone headset](https://en.wikipedia.org/wiki/Acoustic_coupler) with the user dialing separately in advance) and they should communicate. Same with 86Box-es on both ends.

Variations: real modem (or Raspberry Pi/etc.) on the COM port (or via Telnet/SSH/IP) of a computer with 86Box connected to landline (real, [DIY](https://www.vogons.org/viewtopic.php?t=106066) or VoIP)

Related:
- https://github.com/86Box/86Box/discussions/6472#discussioncomment-14975480

**Comment by beyerit** [comment]
I would love this too!

**Comment by Epictyphlosion** [comment]
This idea makes me think of how WinUAE and MAME can play samples of floppy drives when the emulated disk is being accessed. Not sure how implementation of that for dial up sounds would work, but it's a nice idea.

**Comment by artemchik12** [comment]
unfortunately, i lost access to the account i made this suggestion with, it can be closed
